alerta-covid/Gemfile

```diff
@@ -53,6 +53,7 @@ group :development do
   gem 'listen', '~> 3.3'
   # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
   gem 'spring'
+  gem 'rspec-rails'
```

Comment:
You added this gem, but you aren't using it. If you don't plan to use will be better remove it.